src/runtime/arc.ts

```diff
@@ -351,11 +349,20 @@ export class Arc implements ArcInterface {
     return this.idGenerator.newChildId(this.id, component);
   }
 
-  get stores(): Store<CRDTTypeRecord>[] {
-    const stores = [...this.storesByKey.values()];
-    assert(stores.length === Object.keys(this.storeInfoById).length);
-    assert(stores.every(s => !!this.storeInfoById[s.id]));
-    return stores;
+  get stores(): StoreInfo<Type>[] {
+    return Object.values(this.storeInfoById);
+  }
+
+  findActiveStoreById(id: string): Store<CRDTTypeRecord> {
```

Comment:
Particularly now that Stores are StoreInfo instead, I think we should consider renaming ActiveStore to Store. What does Kotlin do here at the moment?

Reply:
they are ActiveStore in KT. I could rename in both places. atm i haven't fully gotten rid of Store, this will happen in next PR.
somewhat related - i'm thinking to rename StoreInfo to StoreOptions to be consistent with KT, WDYT which one is better?

Reply:
I don't have a strong opinion on either name, but consistency is best, and changing typeScript is easier than Kotlin ;-)